src/ru/geekbrains/java/level2/task/ChatMain.java

```diff
@@ -0,0 +1,32 @@
+package ru.geekbrains.java.level2.task;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class ChatMain extends Application {
+    Parent root1;
+    Parent root2;
+    Scene scene1;
+    Scene scene2;
+    @Override
+    public void start(Stage primaryStage) throws Exception{
+        root1 = FXMLLoader.load(getClass().getResource("ChatWindow.fxml"));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.